apps/api/src/models/schema.ts

```diff
@@ -44,6 +46,45 @@ export const friendships = mysqlTable('friendship', {
   // TODO: prevent duplicate rows with inviterId and inviteeId swapped
 }));
 
+export const items = mysqlTable('item', {
+  id: int('id').primaryKey().autoincrement(),
+  createdAt: timestamp('created_at').notNull().$defaultFn(() => new Date()),
+  updatedAt: timestamp('updated_at').onUpdateNow(),
+  creatorId: int('creator_id').references(() => users.id, { onDelete: 'set null' }),
+  description: varchar('text', { length: 1000 }).notNull().unique(),
+  imageUrl: varchar('image_url', { length: 255 }),
+  brand: varchar('brand', { length: 50 }),
+  price: int('price').notNull(), // in cents
+  isForSale: boolean('is_for_sale').notNull().default(false),
+  isForTrade: boolean('is_for_trade').notNull().default(false),
+  soldAt: timestamp('sold_at'),
+});
+
+export const itemsToUsersCollections = mysqlTable('item_to_user_collection', {
+  itemId: int('item_id').unique().notNull().references(() => items.id, { onDelete: 'cascade' }),
+  userId: int('user_id').notNull().references(() => users.id, { onDelete: 'cascade' }),
```

Comment:
Here `.unique()` is used as an item is an _instance_ of some object and hence can only have a single owner.

Currently this is the only difference between Collection and Wishlist Items.